### ISSUE TITLE
Option to display in Dock

### ISSUE THREAD

**Issue by joshuakaufman** [sketchpacks/feedback#12]
I would love if there's an option to display Sketchpacks in the Dock.

Does Sketchpacks really need to run all the time and take up valuable menubar space?

**Comment by jareko999** [comment]
Agreed. Would like to see as a standalone app or at least be able to move it around and resize it.
